portal/src/graphql/portal/LoginMethodConfigurationScreen.tsx

```diff
@@ -2165,6 +2166,19 @@ function VerificationSettings(props: VerificationSettingsProps) {
     [setState]
   );
 
+  const onChangeDailySMSLimit = useCallback(
+    (_, value) => {
+      setState((prev) =>
+        produce(prev, (prev) => {
+          prev.smsRatelimitConfig.per_phone ??= {};
+          prev.smsRatelimitConfig.per_phone.size =
+            parseIntegerAllowLeadingZeros(value);
+        })
+      );
+    },
+    [setState]
+  );
```

Comment:
I suggest we change the shape of the state to match the UI. That is, I will use the following shape as the state.

```typescript
interface State {
  // The valid values are empty string and integers in string form.
  dailySMSSendLimit: string;
}
```

And then I will convert the state in `constructConfig`

```typescript
if (dailySMSSendLimit === "") {
  per_phone = {
    enabled: false
  };
} else {
  per_phone = {
    enabled: true,
    size: parseInt(dailySMSSendLimit, 10),
    reset_period: "24h",
  }
}
```